src/App.vue

```diff
@@ -448,7 +448,7 @@ export default class App extends Mixins(DateMixin, FilingTemplateMixin, LegalApi
         this.initEntityFees()
 
         // Set the resources
-        this.setCertifyStatementResource(CertifyStatementResource.find(x => x.entityType === this.entityType))
+        this.setCompanyResources(CompanyResources.find(x => x.entityType === this.entityType))
       } catch (error) {
```

Comment:
Make a note to review the error handling here. It looks gross (might be my old code), and I'm not certain if it works correctly if, say, an unknown entity type is used.

Reply:
Ok, will do 👍

Reply:
Put in some error handling 👍